workspaces/adr/plugins/adr-backend/src/search/DefaultAdrCollatorFactory.ts

```diff
@@ -51,6 +51,7 @@ import { AuthService, LoggerService } from '@backstage/backend-plugin-api';
 /**
  * Options to configure the AdrCollatorFactory
  * @public
+ * @deprecated Import from `@backstage-community/search-backend-module-adr` instead
  */
 export type AdrCollatorFactoryOptions = {
```

Comment:
Since we're going with the deprecation route now I recommend directly importing these moved types from the `adr-common` package and re-exporting them here so they don't accidentally diverge if a future change mistakenly only updates them in one location before we fully deprecate and remove this copy

Reply:
The type could be easily exported by adding a `export type { AdrCollatorFactoryOptions };`. However, I'm not sure how to re-export the class. The only way is exporting it as a type, which breaks the usage at all.

Reply:
We can follow a pattern found in other packages like https://github.com/backstage/backstage/blob/master/plugins/home/src/deprecated.ts and create a `deprecated.ts` file here to re-export the moved symbols

Reply:
Done. There was a tiny issue due to the export I was using for the class. Now it's solved and the `deprecated.ts` file contains all the deprecated things :)